dolphinscheduler-server/src/main/resources/config/install_config.conf

```diff
@@ -111,6 +111,9 @@ sudoEnable="true"
 # worker tenant auto create
 workerTenantAutoCreate="false"
 
+# worker task plugin dir
+taskPluginDir="/data1_1T/dolphinscheduler/lib/plugin/task"
```

Comment:
It's better to change to relative directory

Reply:
actually, this path referenced the other default value, like 
- alertPluginDir="/data1_1T/dolphinscheduler/lib/plugin/alert" 
- installPath="/data1_1T/dolphinscheduler"

so, after merging this PR , i will raise another PR to fix them all.
What do you think of that?

Reply:
@CalvinKirs What about that？